WooCommerce/Classes/ViewRelated/Orders/Order Creation/PaymentSection/OrderPaymentSection.swift

```diff
@@ -234,9 +228,17 @@ private extension OrderPaymentSection {
     }
 
     @ViewBuilder var taxBasedOnLine: some View {
-        Text(viewModel.taxBasedOnSetting?.displayString ?? "")
-            .footnoteStyle()
-            .multilineTextAlignment(.leading)
+        HStack(spacing: Constants.taxBasedOnLineTextPadding) {
+            Text(viewModel.taxBasedOnSetting?.displayString ?? "")
+                .footnoteStyle()
+                .multilineTextAlignment(.leading)
+            Text(Localization.taxInformationLearnMore)
+                .font(.footnote)
+                .foregroundColor(Color(.wooCommercePurple(.shade60)))
+                .accessibilityLabel(Localization.taxInformationAccessibilityLabel)
```

Comment:
Question: Does a text still require accessibilityLabel?

If I see here: https://developer.apple.com/documentation/swiftui/view/accessibilitylabel(_:)-5f0zj

> Use this method to provide an accessibility label for a view that doesn’t display text, like an icon.

Reply:
Good catch! It won't be necessary anymore since we've changed the icon for a String 👍